script/Params.s.sol

```diff
@@ -2,18 +2,35 @@
 pragma solidity 0.8.25;
 
 contract Params {
-  struct DeploymentParams {
+  struct BFactoryDeploymentParams {
     address bLabs;
   }
 
-  /// @notice Deployment parameters for each chain
-  mapping(uint256 _chainId => DeploymentParams _params) internal _deploymentParams;
+  struct BCoWFactoryDeploymentParams {
+    address bLabs;
+    address settlement;
+    bytes32 appData;
+  }
+
+  /// @notice Settlement address
+  address internal constant _GPV2_SETTLEMENT = 0x9008D19f58AAbD9eD0D60971565AA8510560ab41;
+
+  /// @notice AppData identifier
+  bytes32 internal constant _APP_DATA = bytes32('appData');
+
+  /// @notice BFactory deployment parameters for each chain
+  mapping(uint256 _chainId => BFactoryDeploymentParams _params) internal _bFactoryDeploymentParams;
+
+  /// @notice BCoWFactory deployment parameters for each chain
+  mapping(uint256 _chainId => BCoWFactoryDeploymentParams _params) internal _bCoWFactoryDeploymentParams;
 
   constructor() {
     // Mainnet
-    _deploymentParams[1] = DeploymentParams(address(this));
+    _bFactoryDeploymentParams[1] = BFactoryDeploymentParams(address(this));
```

Comment:
we can store `address(this)` in a `_B_LABS` constant to be set once

Reply:
Shouldn't `blabs` be a a runtime parameter? in that way the deployment can be run with keys with a lower security requirements than the one that'll have the `bLabs` role, e.g. `bLabs` could be a multisig and the contracts be deployed by a random dev with some ether